Search a Sorted Array Of Unknown Size.py

```diff
@@ -0,0 +1,34 @@
+#Time Complexity : O(log(n))
+#Space Complexity : O(1)
+#Runs on my machine
+#Couldn't reduce the search space using the bruteforce, so had to refer to the Sai's solution
```

Comment:
Please try to run the solution of leetcode. Specify if it requires premium to run or not.